src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -84,6 +84,7 @@
 import stream.flarebot.flarebot.analytics.GuildCountAnalytics;
 import stream.flarebot.flarebot.api.ApiRequester;
 import stream.flarebot.flarebot.api.ApiRoute;
+import stream.flarebot.flarebot.api.GzipRequestInterceptor;
```

Comment:
Yegh that class still doesn't exist

Reply:
"Still" I was I need bed.

Reply:
I say still because traivs is falling because of it